janus_server/src/trace.rs

```diff
@@ -47,19 +56,73 @@ pub fn install_subscriber(config: &TraceConfiguration) -> Result<(), Error> {
     // structures
     let output_json = atty::isnt(Stream::Stdout) || config.force_json_output;
 
+    // Filters to prevent verbose runtime-related events from being printed to stdout.
+    // Unfortunately, we need to construct three separate per-layer filters for the different
+    // stdout layers, because their Not types vary depending on their location in the subscriber
+    // stack.
+    let inverted_tokio_console_filter_1 = Targets::new()
+        .with_target("tokio", tracing::Level::TRACE)
+        .with_target("runtime", tracing::Level::TRACE)
+        .not();
+    let inverted_tokio_console_filter_2 = Targets::new()
+        .with_target("tokio", tracing::Level::TRACE)
+        .with_target("runtime", tracing::Level::TRACE)
+        .not();
+    let inverted_tokio_console_filter_3 = Targets::new()
+        .with_target("tokio", tracing::Level::TRACE)
+        .with_target("runtime", tracing::Level::TRACE)
+        .not();
```

Comment:
nit: I think you could factor this into a function and then call `.with_filter(make_console_target())` in the `match` block below.